docs/source/tools/non-repudiation.rst

```diff
@@ -0,0 +1,78 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Non-repudiation
+###############
```

Comment:
We should probably copy the early access & EE disclaimer from the profiler here as well. I know you mention it below but it seems nice to be consistent and have this in the same place everywhere.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/9288/commits/2b27aa7ed0ab4cbbe2db4e2d156811532553c6a5